tests/test_index.py

```diff
@@ -1312,8 +1312,21 @@ def test_multi_index_load_from_directory_2():
     assert len(sigs) == 7
 
 
+def test_multi_index_load_from_directory_3_simple_bad_file(runtmp):
+    # check that force=False fails properly when confrunted with non-JSON
```

Comment:
```suggestion
    # check that force=False fails properly when confronted with non-JSON
```

unless you're being cheeky :)